### PR TITLE
Add padding-around-describe-blocks

### PR DESCRIPTION
**Pull request by benkimpel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,10 +61,9 @@ _or_
 - [padding-around-before-all-blocks](docs/rules/padding-around-before-all-blocks.md)
 - [padding-around-before-each-blocks](docs/rules/padding-around-before-each-blocks.md)
 - [padding-around-expect-statements](docs/rules/padding-around-expect-statements.md)
+- [padding-around-describe-blocks](docs/rules/padding-around-describe-blocks.md)
 
-- [padding-before-describe-blocks](docs/rules/padding-before-describe-blocks.md)
 - [padding-before-test-blocks](docs/rules/padding-before-test-blocks.md)
-
```
Comment (benkimpel) [inline README.md]: Just a little blank line clean up here. Markdown was formatting the main list with extra space.

File: tests/lib/rules/padding-around-describe-blocks.spec.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
+/**
```
Comment (benkimpel) [inline tests/lib/rules/padding-around-describe-blocks.spec.js]: It turns out these tests didn't need to change to check the around functionality. hooray.

File: tests/lib/rules/padding-around-describe-blocks.spec.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+ * @fileoverview Enforces single line padding around describe blocks
+ * @author Dan Green-Leipciger
```
Comment (benkimpel) [inline tests/lib/rules/padding-around-describe-blocks.spec.js]: I dunno if you have a preference on `@author` or not with this stuff. Personally, I'm fine leaving me off of stuff. Lemme know... this was in the one that I copy/pasted from.

File: tests/lib/rules/padding-around-describe-blocks.spec.js (hunk 2 above)
Comment (benkimpel) [reply tests/lib/rules/padding-around-describe-blocks.spec.js]: Nope, that can't be true. I think i need to add a statement after a describe. one sec.

Comment (benkimpel) [reply tests/lib/rules/padding-around-describe-blocks.spec.js]: Yes, to test the after I needed something other than a `describe` following a `describe`.

File: tests/lib/rules/padding-around-describe-blocks.spec.js (hunk 3 above)
Comment (dangreenisrael) [reply tests/lib/rules/padding-around-describe-blocks.spec.js]: I'm easy. You can put your name there. You can also just remove it. Up to you

Comment (benkimpel) [reply tests/lib/rules/padding-around-describe-blocks.spec.js]: Let's make up a third person.

**Pull request merged**